dataflow/flex-templates/streaming_beam/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM gcr.io/dataflow-templates-base/python3-template-launcher-base
+
+RUN mkdir -p /cloud/dataflow
```

Comment:
I would suggest using `WORKDIR`. Also from a branding perspective, Rose let us know that Dataflow is dropping the "Cloud", so now it's just "Dataflow" instead of "Cloud Dataflow", so maybe it's good to make the workdir reflect that.

We could do `/dataflow/template` or just `/template`.

```
ARG WORKDIR=/dataflow/template
RUN mkdir -p ${WORKDIR}
WORKDIR ${WORKDIR}

COPY streaming_beam.py .
...
ENV FLEX_TEMPLATE_PYTHON_REQUIREMENTS_FILE="${WORKDIR}/requirements.txt"
...
```

Reply:
Done.